src/blockchain_utilities/blockchain_blackball.cpp

```diff
@@ -525,6 +526,7 @@ int main(int argc, char* argv[])
           state.spent.insert(output_data(od.amount, last_unknown));
```

Comment:
Can't this also be dropped?

Reply:
It could, but these rounds use very little memory.